build.gradle

```diff
@@ -108,6 +108,14 @@ subprojects {
             }
         }
 
+        ['ida_utils','verify_saml','dropwizard','dropwizard_assets'].each {
+            configurations.getByName(it) {
```

Comment:
What's the `it` referring to?

Reply:
I think it's the each of the eaches. 😆 Or an implicit parameter.

https://groovy-lang.org/closures.html

One day I'm going to do a project and use Groovy for the testing because there'a s really cool library the name of which I've forgotten but it looked great and used all of those groovy features.

Reply:
This is the library: https://www.baeldung.com/groovy-spock